l10n_it_fatturapa_pec/__init__.py

```diff
@@ -1,3 +1,3 @@
 
-from . import models
-from . import wizard
+from . import models  # noqa
+from . import wizard  # noqa
```

Comment:
Come mai `noqa`?

Reply:
evita di mostrare l'errore imported but not used

Reply:
Se fosse così, bisognerebbe farlo per tutti gli `__init__` di tutti i moduli

Reply:
Sarebbe cosa buona e giusta, anche se in realtà è un errore "tollerato" dalle guidelines oca.

Io lo faccio, come dicevo, per evitare che l'ide segnali l'errore.

Reply:
Il punto è che in tutti gli altri moduli non viene fatto, quindi non va fatto neanche qua

Reply:
Qua vedo ancora `# noqa`